app/scripts/behaviors/wsreader-behavior.js

```diff
@@ -1,21 +0,0 @@
-var Behaviors = Behaviors || {};
-
-Behaviors.WSReaderBehavior = {
-	listeners: {
-		'ws-onerror': '_onWsError',
-		'ws-onmessage': '_onWsMessage',
-		'ws-onopen': '_onWsOpen'
-	},
-	_onWsOpen: function(){
-		// default action
-		console.log('WS OPENED');
```

Comment:
it is pointless because that methods need to be implemented anyway because they depend on the context.